src/index.ts

```diff
@@ -99,7 +99,7 @@ export
         cellTagRoot = null;
       }
     }
-    else if (cellTagRoot === null) {
```

Comment:
@ricklamers For first-time user (i.e. `tags.length === 0`), this function ends at line 100 and thus nothing happens afterwards. Removing the `else` ensures that the rest of the function is executed.